### ISSUE TITLE
Test LSP objects

### ISSUE THREAD

**Issue by fm-117** [TypeCobolTeam/TypeCobol#2659]
Our implementation of LSP is quite old now and more importantly is embedded instead of relying on a well-known externally-provided package.

Migrating to `Microsoft.VisualStudio.LanguageServer.Protocol` NuGet package is the target however this will be difficult as we have accumulated a lot of gaps over time.

The goal of this issue is to create tests covering the form of protocol objects. This will allow:
- to identify discrepancies between our private client definitions and server definitions
- to identify discrepancies between multiple version of the protocol objects (when migrating to VS package)

In a nutshell the protocol test should:
- automatically discover the messages known by this server
- verify that each message is actually covered by a test
- test every message using predefined JSON test files
